cluster/gce/gci/configure-helper.sh

```diff
@@ -675,14 +717,27 @@ contexts:
     user: kube-apiserver
   name: webhook
 EOF
-    # This is the config for the image review admission controller.
+    GCP_IMAGE_REVIEW_PLUGIN="- name: ImagePolicyWebhook
+  configuration:
```

Comment:
This "name/configuration" level seems like it wasn't present before, did the image policy webhook format change? Or has this just been broken since @deads2k's config file format change?

Reply:
I initially had it preserving the functionality exactly as it was before, until [this discussion](https://github.com/kubernetes/kubernetes/pull/55907#discussion_r155681193). I think the config file has been broken for a while